src/utils/conditional-tag.ts

```diff
@@ -0,0 +1,56 @@
+export function parseConditionalTag(tag: string) {
```

Comment:
This seems like some fairly complex logic which I don't follow without some study. Could we add unit tests to the project and some basic tests for this?

I'd recommend Jest, I'm very happy to create a PR with a basic Jest configuration and example test.

Reply:
It will be wonderful

Reply:
Added in PR https://github.com/zlant/parking-lanes/pull/97. Please merge that then rebase this branch on master so we can add tests for this code.

Reply:
Ready